### DISCUSSION TITLE
Result inconsistent

### DISCUSSION THREAD

**Discussion by riyajatar93** [open-mmlab/mmocr#1995] [General]
i am doing inference with below code , it return detection and recognized text.

infer = MMOCRInferencer(det='dbnetpp', rec='svtr-small')

is there any way i can get output in json format like the one 
ocr.readtext gives.

also order of text detected on input image is not consistent , i am unable to sort them properly . any method or suggestion to solve this
thanks

**Comment by gaotongxiao** [comment]
The output structure has been significantly refactored and aligned with the global design of OpenMMLab, and therefore you might need a custom postprocessor to achieve this effect.

Currently there is no universal box/text sorting algorithm, but MMOCR has one implementation that you might be interested in:

https://github.com/open-mmlab/mmocr/blob/b18a09b2f063911a2de70f477aa21da255ff505d/mmocr/utils/bbox_utils.py#L131
